backend/FixFwData/FixFwData.csproj

```
@@ -2,14 +2,17 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+    <RootNamespace>FixFwData</RootNamespace>
+    <Description>FixFwData</Description>
+    <Company>SIL International</Company>
+    <Authors>SIL International</Authors>
+    <Product>LexBoxApi Testing</Product>
+    <Copyright>Copyright © 2023 SIL International</Copyright>
```

I'm not sure if these tags were added on purpose, but if we keep them they should be:
```suggestion
    <Company>SIL Global</Company>
    <Authors>SIL Global</Authors>
    <Product>LexBoxApi Testing</Product>
    <Copyright>Copyright © 2024 SIL Global</Copyright>
```